### PR TITLE
Fix docs-update action

### PR DESCRIPTION
**Pull request by pamella**
## Issues

#119

## Description

Fix broken action. See failed run: https://github.com/vintasoftware/django-ai-assistant/actions/runs/9686227984/job/26728174292

image

### CODE REVIEW

File: .github/workflows/release.yml
Comment (pamella) [inline .github/workflows/release.yml:1]: Drive-by change. Removes unnecessary step, it was used for debugging purposes.

**Review by fjsj — APPROVED** [review]
